src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -1202,9 +1258,9 @@ public void testPutOrdersByIdToChangeStatusToOpenWithEmptyPoLines() throws Excep
   private void verifyReceiptStatusChangedTo(String expectedStatus) {
     List<JsonObject> polUpdates = MockServer.serverRqRs.get(PO_LINES, HttpMethod.PUT);
     assertNotNull(polUpdates);
-    // check with filter last 2 putted polines
+    // check receipt status for the last 2 putted polines
```

Comment:
"check receipt status of the last 2 updated polines"

Reply:
fixed